docs/topics/js/js-interop.md

```diff
@@ -202,12 +202,36 @@ fun usingAsOperator(s: Any) = s as String
 They will be compiled accordingly:
 
 ```javascript
-function usingUnsafeCast(s) {
+function usingUnsafeCast (s) {
     return s;
 }
 
-function usingAsOperator(s) {
+function usingAsOperator (s) {
     var tmp$;
     return typeof (tmp$ = s) === 'string' ? tmp$ : throwCCE();
 }
-```
\ No newline at end of file
+```
+
+## Equality
+
+Kotlin/JS has particular semantics for equality comparison, especially in the case of strings. 
+
+In Kotlin/JS, there is no difference between `===` and `==` operators when comparing strings. The reason is that in Kotlin/JS,
+the `==` operator checks that two values are equal. Meanwhile, the `===` operator checks both that two values are equal 
+and that the types of these two values are also equal. In JS, Kotlin always uses the `===` operator:
```

Comment:
I believe here we mixed Kotlin and JS operators.
It's better to say that in K/JS, Kotlin === operator always translates to JS === operator, which checks that types and values are equal.

Reply:
Does Kotlin === operator always translates to JS === operator when comparing strings or can I remove "when comparing strings"?
I rephrased it, please take a look.

Reply:
As far as I can tell, Kotlin === always translates to JS === for all operand types.